hbase-server/src/main/java/org/apache/hadoop/hbase/snapshot/SnapshotDescriptionUtils.java

```diff
@@ -465,4 +465,18 @@ public ListMultimap<String, UserPermission> run() throws Exception {
     return snapshot.toBuilder()
       .setUsersAndPermissions(ShadedAccessControlUtil.toUserTablePermissions(perms)).build();
   }
+
+  /**
+   * Method to check whether TTL has expired for specified snapshot creation time and snapshot ttl.
+   * NOTE: For backward compatibility (after the patch deployment on HMaster), any snapshot with ttl
+   * 0 is to be considered as snapshot to keep FOREVER. Default ttl value specified by
+   * {@link HConstants.DEFAULT_SNAPSHOT_TTL}
```

Comment:
I doubt if javadoc might recognize it, I think only `{@link HConstants#DEFAULT_SNAPSHOT_TTL}` will work mostly

Reply:
Ah my bad, let me fix